config/rca.conf

```diff
@@ -65,6 +65,14 @@
     "shard-request-cache-rca": {
       "shard-request-cache-threshold" : 0.9
     },
+    "search-back-pressure-rca": {
+       "max-heap-usage-increase": 80,
+       "max-shard-heap-cancellation-percentage": 5,
```

Comment:
Again I am guessing this is defined 5 as per thresholds defined at OpenSearch level?

Reply:
These are based on the rate limiters on total cancellation percentage w.r.t. total request count. Since the max we can cancel is 10% due to all resources (CPU, MEM, TIME). I have kept it as 5 given we have some heap left. But if heap is still in red this can still go ahead and cancel the in-flight tasks.